server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -312,9 +314,21 @@ export class ScribeLambda implements IPartitionLambda {
                     // An external summary writer can only update the protocolHead when the ack is sequenced
                     // back to the stream.
                     if (this.summaryWriter.isExternal) {
-                        this.protocolHead = content.summaryProposal.summarySequenceNumber;
+                        await this.updateProtocolHead(content.summaryProposal.summarySequenceNumber);
                     }
                 }
+
+                // check to see if this exact sequence number causes us to hit the max ops since last summary nack limit
+                if (this.serviceConfiguration.scribe.nackMessages.enable &&
+                    this.serviceConfiguration.scribe.nackMessages.maxOps === this.sequenceNumber - this.protocolHead) {
```

Comment:
The reason its "===" and not "<=" is because we only want to send it once? Can deli accept duplicate control message?

Reply:
It can accept duplicate but it really only needs to send it once since it remembers. It would be a waste to keep sending it for every op over the limit